src/ADAL.PCL.iOS/Properties/AssemblyInfo.cs

```diff
@@ -39,7 +39,4 @@
 // The following GUID is for the ID of the typelib if this project is exposed to COM
 [assembly: Guid("ff47962a-d498-4c63-b7e9-4db3653ad7e0")]
 
-// Assembly version information is in file ADAL.Common\CommonAssemblyInfo.cs
-
-[assembly: AssemblyDelaySign(true)]
-[assembly: AssemblyKeyFile(@"..\..\build\35MSSharedLib1024.snk")]
```

Comment:
we require delay signing for our libraries. any specific reason this was removed?